usim/_primitives/context.py

```diff
@@ -135,10 +136,20 @@ def do(
             self._volatile_children.append(child_task)
         return child_task
 
+    def __cancel__(self):
+        """Cancel this scope"""
+        __LOOP_STATE__.LOOP.schedule(self._activity, self._cancel_self)
+
     async def _await_children(self):
         for child in self._children:
             await child.done
 
+    async def _reraise_children(self):
+        for child in self._children:
+            print('inspect', child)
```

Comment:
Is it intentional that you do a `print` here? If you want to output something you should prefer logging.

Reply:
Whoops, that wasn't intentional. Fixed.